src/api/app/components/bs_request_overview_avatars_component.rb

```diff
@@ -26,7 +26,12 @@ def group_avatar_objects
   end
 
   def package_avatar_objects
-    [@review.package&.project&.users].flatten.compact
+    reviewers = @review.maintainers_and_reviewers_for_package
+    # if the package explicitly has maintainer or reviewer assigned return them, otherwise
+    # check the related project for responsible user
+    return reviewers if reviewers.present?
+
+    @review.maintainers_and_reviewers_for_project
```

Comment:
Is it possible there are no reviewers or maintainers present for a project?

Reply:
It is possible, yes

Reply:
In that case, we can return users who are not reviewers or maintainers.

Reply:
@rubhanazeem I would not return anything else than reviewers or maintainers. If both package and project don't have any reviewer or maintainers assigned there is no one who could review. To me thats a sign that the package or project is not maintained anymore. So just showing the reference to the package is fine IMO